test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,38 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
These tests can be separated into multiple more specific methods. Refer to https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html for our standard of naming the methods.